core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -204,16 +203,24 @@ sealed class Vec[T <: Data] private[chisel3] (gen: => T, val length: Int) extend
     // Since all children are the same, we can just use the sample_element rather than all children
     direction =
       ActualDirection.fromChildren(Set(sample_element.direction), resolvedDirection).getOrElse(ActualDirection.Empty)
+    this.maybeAddToParentIds(target)
```

Comment:
i dont understand this much, but should it still be called `maybe` after this change?